src/routing/comm.rs

```diff
@@ -292,6 +296,87 @@ impl SendState {
     }
 }
 
+// Stream of incoming messages from all existing and incoming connections.
```

Comment:
`from all existing and incoming connections.` ?
it's not for just one connection?

the constructor seems to construct new channel for new incoming_conns ?

Reply:
OK, that wording is clumsy, but what I wanted to say is that it will receive messages on all established connections including the ones that will only be established in the future. In other words, it listen for incoming connections and then multiplex the messages from those connection into a single stream. Will try to reword that.